main.tf

```diff
@@ -0,0 +1,7 @@
+resource "serverless_deployment" "example" {
```

Comment:
Move to example if we can?

Reply:
Should/can we rename it to `example-test.tf` or something to clearly distinguish it?

Reply:
Done!